internal/collector/vector/visitors.go

```diff
@@ -44,6 +45,11 @@ func CollectorVisitor(collectorContainer *corev1.Container, podSpec *corev1.PodS
 			Items:      []corev1.KeyToPath{{Key: RunVectorFile, Path: RunVectorFile}},
 			Optional:   utils.GetPtr(true),
 		}}},
+		corev1.Volume{Name: common.SecretDataReader, VolumeSource: corev1.VolumeSource{Secret: &corev1.SecretVolumeSource{
```

Comment:
This is no longer a "SecretVolumeSource now that you are creating a configmap

Reply:
Rethinking... to clarify:
* the "data" or our vault must be mounted placed in a secret and mounted as a secret volume
* the config can go in a configmap and is mounted as a configmpa volume

I can help with some of this path mounting since I have to work on how the CW token projections are integrated here